docs/_basic/ja_web_api.md

```diff
@@ -17,7 +17,7 @@ Bolt アプリケーションは、トップレベルに `app.client` も持っ
 
 ```javascript
 // September 30, 2019 11:59:59 PM を Unix エポックタイムで表示
-const whenSeptemberEnds = 1569887999;
+const whenSeptemberEnds = '1569887999';
```

Comment:
indeed, the value is a `string` in the TypeScript world. https://github.com/slackapi/node-slack-sdk/blob/%40slack/web-api%406.1.0/packages/web-api/src/methods.ts#L982